### PR TITLE
Support colored text while reporting

### PR DESCRIPTION
**Pull request by pingsutw**
`\033[91m` for failure testing (red)
`\033[0m` for resetting color (white)
`\033[92m` for successful testing (green)

### CODE REVIEW

**Comment by jserv** [comment]
Considering the use case of continuous integration (CI), we might expect the output without color. So, please think of feasible designs to allow switching between colorfulness and monocolor.

**Comment by pingsutw** [comment]
@jserv, Thanks for the reply and suggestion
Add `-c` option in scripts/driver.py, '-c' means colorful
By default is monocolor

File: scripts/driver.py [hunk 1]
```diff
@@ -14,6 +16,7 @@ class Tracer:
     verbLevel = 0
     autograde = False
     useValgrind = False
+    colorFul = False
```
Comment (jserv) [inline scripts/driver.py]: Simply use `colored`.

File: scripts/driver.py [hunk 2]
```diff
@@ -124,11 +144,12 @@ def run(self, tid=0):
 
 
 def usage(name):
-    print("Usage: %s [-h] [-p PROG] [-t TID] [-v VLEVEL] [--valgrind]" % name)
+    print("Usage: %s [-h] [-p PROG] [-t TID] [-v VLEVEL] [--valgrind] [-c]" % name)
     print("  -h        Print this message")
     print("  -p PROG   Program to test")
     print("  -t TID    Trace ID to test")
     print("  -v VLEVEL Set verbosity level (0-3)")
+    print("  -c Print colored text in terminal")
```
Comment (jserv) [inline scripts/driver.py]: Change the message to `Enable colored text`.

Comment (pingsutw) [reply scripts/driver.py]: Done

File: scripts/driver.py (hunk 1 above)
Comment (pingsutw) [reply scripts/driver.py]: Done

**Comment by pingsutw** [comment]
Thanks for the review

**Pull request merged**